src/lib/functions/registry.ts

```diff
@@ -453,34 +447,27 @@ export class FunctionsRegistry {
   /**
    * A proxy to zip-it-and-ship-it's `listFunctions` method. It exists just so
    * that we can mock it in tests.
-   * @param  {Parameters<listFunctions>} args
-   * @returns
    */
-  // @ts-expect-error TS(7019) FIXME: Rest parameter 'args' implicitly has an 'any[]' ty... Remove this comment to see the full error message
   // eslint-disable-next-line class-methods-use-this
```

Comment:
Can we kill this lint rule? 🗡️

Reply:
You mean the `class-methods-use-this` rule? Yeah, maybe we should 😅